### PR TITLE
Dark mode

### PR DESCRIPTION
**Pull request by andylib93**
Here is my first try for dark mode support. I haven't got the experience to get docker running quickly enough, so I wasn't able to set the concrete values for the dark mode implementation. Yet I found the GitHub Darkmode Dimmed theme and added the values as comments to the css block where they would be set. Additionally I had trouble naming the variables for the markdown section of your css code - perhaps you could take a look at that?
So in short: it's not done yet, but the fundamentals are set to be further implemented 😊️ oh and I added a bunch of missing semicolons, I don't know if that was intentional?

### CODE REVIEW

File: internal/web/playground.css [hunk 1]
```diff
@@ -1,17 +1,66 @@
...
+    --dropdown-border: #a5a5a5;
+    --dropdown-hover-background: #375eab;
+    --dropdown-selected-background: #1f3663;
+
+    --footer-color: #646262;
+}
+
+@media (prefers-color-scheme: dark){
+    :root {
+        --toolbar: #000;
```
Comment (feliixx) [inline internal/web/playground.css:37]: this is not used anywhere, did you mean `--toolbar-color` ?

**Review by feliixx — COMMENTED** [review]
This looks great ! Picking variables names is always the hardest part, these are good enough

No idea why some semicolons were missing, thanks for fixing it 
You can test your change without docker like this: 

```
make some change to **playground.css**
./bundle.sh   
go build 
./mongoplayground
```
thanks again for working on this

File: internal/web/playground.css (hunk 1 above)
Comment (andylib93) [reply internal/web/playground.css:37]: Oh yes, my mistake 👋🏻

**Comment by andylib93** [comment]
Ah okey nice, I will try that in the coming days 😎
And no problem, glad I could be of help 😇
